backtester/eventhandlers/statistics/printresults.go

```diff
@@ -121,10 +122,10 @@ func (s *Statistic) PrintAllEventsChronologically() {
 			log.Info(common.Statistics, results[i].Events[j])
 		}
 	}
-	if len(errs) > 0 {
+	if errs != nil {
 		log.Info(common.Statistics, common.CMDColours.Error+"------------------Errors-------------------------------------"+common.CMDColours.Default)
-		for i := range errs {
-			log.Error(common.Statistics, errs[i].Error())
+		for err := errors.Unwrap(errs); err != nil; err = errors.Unwrap(errs) {
+			log.Error(common.Statistics, err.Error())
```

Comment:
This will duplicate print errors. a wrapped errs using err.Error() will print the comma delimited list. So for each error within errs, will print the whole list.
See this test:
```go
func TestLoop(t *testing.T) {
	t.Parallel()
	errOne := errors.New("one")
	errTwo := errors.New("two")
	errThree := errors.New("three")
	err := AppendError(errOne, errTwo)
	errs := AppendError(err, errThree)
	errs = AppendError(errs, errThree)
	for err := errors.Unwrap(errs); err != nil; err = errors.Unwrap(errs) {
		t.Log(err.Error())
	}
}
```
output:
```    
    common_test.go:822: one, two, three, three
    common_test.go:822: one, two, three, three
    common_test.go:822: one, two, three, three
```